aspnetcore/performance/performance-best-practices/samples/3.0/Startup22.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+
+namespace PBP
+{
+    public class Startup22
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            #region snippet1
+            app.Use(async (context, next) =>
+            {
+                await context.Response.WriteAsync("Hello ");
```

Comment:
I'd remove this line too. Even though this is the "bad" example, it's supposed to be bad because we're not checking `context.Response.HasStarted`, not just because we wrote to the response body and then tried changing the response header (which is also bad ofc).